about.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    <title>About</title>
+</head>
+<body>
+    <nav>
+        <div class="logo"><a href="index.html">My Logo</a></div>
+        <ul class="navbar">
+            <li class="nav-items"><a href="index.html">home</a></li>
+            <li class="nav-items"><a href="about.html">about</a></li>
+            <i class="fas fa-bars" onclick="displayMenu()"></i>   
+        </ul>
+        
+    </nav>
+    <div id='popup'></div>
+    <header class="about-header">
+        <h5 class="first-text about-f-text">"Hello!, there"</h5>
+        <h1 class="second-text about-s-text">about us</h1>
+        <p class="paragraph about-p">
+            We as developers saw the neccesity of a powerful software that can help the small businesses to thrive in this
+            everyday more competitive world and also help enterpeneours accomplish they objectives in their business and life
+            that was the engine behind this softare, helping others!.
+        </p>
+        <div class="about-logo">my logo</div>
+    </header>
```

Comment:
The section needs to be styled based on the given design as shown below

#### Original Design
![right3](https://user-images.githubusercontent.com/58376020/124035735-3db29880-d9fd-11eb-9369-0c71043d4c50.png)

#### Your Project
![wrong3](https://user-images.githubusercontent.com/58376020/124035766-499e5a80-d9fd-11eb-9f04-2da9bfcb1c86.png)